client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,8 +44,8 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Support for Camunda 8.6 features</h4>
-          This release adds support for upcoming Camunda 8.6 features, including execution listeners, the new binding type property, and converging inclusive gateways.
+          <h4>Camunda 8 Connector support</h4>
+          Use Camunda 8 Connectors to access a growing range of external services and communication protocols. In the modeler, you can now quickly and easily add pre-built Camunda connectors to your BPMN processes. To learn how to configure this feature, checkout the <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/use-connectors/">connector documentation</a>.
```

Comment:
I'd drop "In the modeler" as this is the release info for the Modeler.

Reply:
Adjusted via https://github.com/camunda/camunda-modeler/pull/4474/commits/29d2c130c657d5be698a6c2b16b1b72e1cb7b568.